sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -467,10 +492,134 @@ The `node` class provides the {crs}[common reference semantics].
 [source,c++]
 ----
 namespace sycl::ext::oneapi::experimental {
-  class node {};
+  class node {
+    template <int Dimensions>
+    void update_nd_range(nd_range<Dimensions> executionRange);
+  };
 }
 ----
 
+Table {counter: tableNumber}. Member functions of the `node` class.
+[cols="2a,a"]
+|===
+|Member Function|Description
+
+|
+[source,c++]
+----
+template <int Dimensions>
+void update_nd_range(nd_range<Dimensions> executionRange);
+----
+| Updates the ND-Range for this node with a new value. This new value will not
+affect any executable graphs this node is part of until it is passed to the
+executable graph's update function.
+See <<executable-graph-update, Executable Graph Update>> for more information
+about updating node parameters.
+
+Parameters:
+
+* `executionRange` - The new value for the ND-Range.
+
+Exceptions:
+
+* Throws with error code `nd_range` if `executionRange` is an invalid ND-Range
+  for this node.
+
+* Throws with error code `invalid` if the type of the node is not a kernel
+  execution.
+
+|===
+
+==== Dynamic Parameters
+
+Dynamic parameters are arguments to a node's command-group which can be updated
+by the user after the node has been added to a graph. Updating the value of a
+dynamic parameter will be reflected in the modifiable graph which contains this
+node. Executable graphs can then be updated with these update nodes
```

Comment:
```suggestion
node.
```
Sentence doesn't make sense to me. Maybe we can be more specific or just remove it.

Reply:
I've tried to rework this sentence to make it read better. I think it's important to keep something here that explains how the update makes it's way to the executable graph in the end though.